src/openrct2/peep/Guest.cpp

```diff
@@ -4126,12 +4168,19 @@ void Guest::UpdateRideLeaveVehicle()
     if (ride == nullptr)
         return;
 
-    Vehicle* vehicle = GET_VEHICLE(ride->vehicles[CurrentTrain]);
+    Vehicle* vehicle = GetEntity<Vehicle>(ride->vehicles[CurrentTrain]);
+    if (vehicle == nullptr)
+        return;
```

Comment:
Just to be consistent with most of the other places.
```suggestion
    if (vehicle == nullptr)
    {
        return;
    }
```

Reply:
I was being consistent within the function.